plugins/query.rst

```diff
@@ -0,0 +1,26 @@
+Query plugin
+==============
+
+Default Query parameters
+----------------------
+
+The plugin ``QueryDefaultsPlugin`` allows you to define default values for
+query parameters. If a query parameter is not set, it will be added. However, if the query parameter
+is already present, the request is left unchanged. Names and values must not be url encoded as this
+plugin will encode them::
+
+    use Http\Discovery\HttpClientDiscovery;
+    use Http\Client\Common\PluginClient;
+    use Http\Client\Common\Plugin\HeaderDefaultsPlugin;
+
+    $defaultUserAgent = 'Mozilla/5.0 (Windows NT 6.1; WOW64; rv:40.0) Gecko/20100101 Firefox/40.1';
```

Comment:
this is unused, i think its from the default header plugin

Reply:
I went too fast. Sorry for this.